stylesheets/_global.scss

```diff
@@ -4,7 +4,7 @@
 
 html {
   height: 100%;
-  border: solid 1px #ccc;
+  border: none;
```

Comment:
I think we simply omit this line entirely. It's an artifact from when we had frameless windows.